### PR TITLE
Add Terminus Font license

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
```SUMMARY: None```

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -4,6 +4,105 @@ The Square font (data/font/square.ttf) is copyright Wouter van Oortmerssen, lice
 
 GNU Unifont (data/font/unitfont.ttf) is licensed under GNU General Public License v2+ with the GNU font embedding exception. Visit http://unifoundry.com/LICENSE.txt for details.
 
+Terminus Font (data/font/terminus.ttf) is licensed under the SIL Open Font License license. The full license text is as follows:
```
Comment (ZhilkinSerg) [inline LICENSE.txt]:
```suggestion
Terminus Font (data/font/terminus.ttf) is licensed under the SIL Open Font License. The full license text is as follows:
```

File: LICENSE.txt [hunk 2]
```diff
@@ -4,6 +4,105 @@ The Square font (data/font/square.ttf) is copyright Wouter van Oortmerssen, lice
...
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
+OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
+DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
+OTHER DEALINGS IN THE FONT SOFTWARE.
+
+***
```
Comment (ZhilkinSerg) [inline LICENSE.txt]:
```suggestion
```

File: LICENSE.txt [hunk 3]
```diff
@@ -4,6 +4,105 @@ The Square font (data/font/square.ttf) is copyright Wouter van Oortmerssen, lice
...
+
```
Comment (ZhilkinSerg) [inline LICENSE.txt]:
```suggestion
```

**Comment by kevingranade** [comment]
Please put the license text in a new file and reference it from license.txt

**Pull request merged**